### ISSUE TITLE
usage experience

### ISSUE THREAD

**Issue by dayAndnight2018** [whistyun/Markdown.Avalonia#38]
Question 1:

![image](https://user-images.githubusercontent.com/44399091/105148026-dda9f300-5b3c-11eb-90a3-9c3f07e52ee2.png)
when I use ">" to perform a break line, it not work. if I add a blank line between ">" lines, it looks ugly.

Question 2:

if it is needed to support padding when I use a “``` to show codes ```” under a list item? otherwise, it looks ugly, too! when I use "```code ```" under a list item, it also blocks the number to be auto increment as what shows in the following picture.

![image](https://user-images.githubusercontent.com/44399091/105148592-86f0e900-5b3d-11eb-870e-6e3ead216bf3.png)

**Comment by whistyun** [comment]
About Q1

This is match to specifications.

If you insert empty line between ">" lines. Each separated ">" lines are displayed in each different paragraphs.

If you want to change the paragraph in blockquote. You should precode empty line with '>'.

**ugly sample**  
```md
> one

> two
```
> one

> two

**correct sample**  
```md
> one
>
> tww
```
> one
>
> tww

**Comment by dayAndnight2018** [comment]
> About Q1
> 
> This is match to specifications.
> 
> If you insert empty line between ">" lines. Each separated ">" lines are displayed in each different paragraphs.
> 
> If you want to change the paragraph in blockquote. You should precode empty line with '>'.
> 
> **ugly sample**
> 
> ```lisp
> > one
> 
> > two
> ```
> 
> > one
> 
> > two
> 
> **correct sample**
> 
> ```lisp
> > one
> >
> > tww
> ```
> 
> > one
> > tww

I want a new line instead of a blank line

**Comment by whistyun** [comment]
If you want an ONE linebreak, you have to add 2 spaces  to line tail.

```
> one  
> two
```

> one  
> two

**Comment by dayAndnight2018** [comment]
> > About Q1
> > This is match to specifications.
> > If you insert empty line between ">" lines. Each separated ">" lines are displayed in each different paragraphs.
> > If you want to change the paragraph in blockquote. You should precode empty line with '>'.
> > **ugly sample**
> > ```lisp
> > > one
> > 
> > > two
> > ```
> > 
> > 
> > > one
> > 
> > 
> > > two
> > 
> > 
> > **correct sample**
> > ```lisp
> > > one
> > >
> > > tww
> > ```
> > 
> > 
> > > one
> > > tww
> 
> I want a new line instead of a blank line

ok, how about question 2?

**Comment by dayAndnight2018** [comment]
> If you want an ONE linebreak, you have to add 2 spaces to line tail.
> 
> ```
> > one  
> > two
> ```
> 
> > one
> > two
> 
> If you want more than one linebreak, I think it should be solved by [#30](https://github.com/whistyun/Markdown.Avalonia/issues/30#issuecomment-753469850).

it not satisfy me about padding at the head of a new line:

![image](https://user-images.githubusercontent.com/44399091/105259899-adf3fd00-5bc7-11eb-8ceb-872ecfdcb300.png)

**Comment by whistyun** [comment]
About Q2

It is bug that codeblock can not  be included in list.

Not support starting ordered lists at an arbitrary number is not specification violation.
But supporting is allowed, so I may support it.

**Comment by whistyun** [comment]
> > If you want an ONE linebreak, you have to add 2 spaces to line tail.
> > ```
> > > one  
> > > two
> > ```
> > 
> > 
> > > one
> > > two
> > 
> > 
> > If you want more than one linebreak, I think it should be solved by [#30](https://github.com/whistyun/Markdown.Avalonia/issues/30#issuecomment-753469850).
> 
> it not satisfy me about padding at the head of a new line:
> 
> ![image](https://user-images.githubusercontent.com/44399091/105259899-adf3fd00-5bc7-11eb-8ceb-872ecfdcb300.png)

Did you mean that you want treat '>' as codeblock?
'>' is blockquote syntax for quottng a sentence, So duplication spaces is removed.

And If you indent a sentence, It is treated as codeblock.
It's strange that it doesn't require a blank line before indentation, but it follows the spec approximately.

**Comment by dayAndnight2018** [comment]
> > > If you want an ONE linebreak, you have to add 2 spaces to line tail.
> > > ```
> > > > one  
> > > > two
> > > ```
> > > 
> > > 
> > > > one
> > > > two
> > > 
> > > 
> > > If you want more than one linebreak, I think it should be solved by [#30](https://github.com/whistyun/Markdown.Avalonia/issues/30#issuecomment-753469850).
> > 
> > 
> > it not satisfy me about padding at the head of a new line:
> > ![image](https://user-images.githubusercontent.com/44399091/105259899-adf3fd00-5bc7-11eb-8ceb-872ecfdcb300.png)
> 
> Did you mean that you want treat '>' as codeblock?
> '>' is blockquote syntax for quottng a sentence, So duplication spaces is removed.
> 
> And If you indent a sentence, It is treated as codeblock.
> It's strange that it doesn't require a blank line before indentation, but it follows the spec approximately.

I think it as the "```"，isn't it?

**Comment by whistyun** [comment]
Did you say about this translation?
![image](https://user-images.githubusercontent.com/13712028/105345785-0529ba00-5c28-11eb-87da-7f2e7f3499df.png)

In Markdown, code block is expressed two syntaxes.
1. surrounded by "```" 
    ````md
    ```
    code
    ```
    ````
    ```
    code
    ```
2. indentation
    ````md
        code
    ````

        code

**Comment by dayAndnight2018** [comment]
> Did you say about this translation?
> ![image](https://user-images.githubusercontent.com/13712028/105345785-0529ba00-5c28-11eb-87da-7f2e7f3499df.png)
> 
> In Markdown, code block is expressed two syntaxes.
> 
> 1. surrounded by "```"
>    ```lisp
>    ```
>    code
>    ```
>    ```
>    
>    
>    ```
>    code
>    ```
> 2. indentation
>    ```lisp
>        code
>    ```
>    
>    
>    ```
>     code
>    ```

what I want is just like this:

```
const(
     A = A,
     B = B
)
```

**Comment by dayAndnight2018** [comment]
another question is : when I use "```" frequently，the parse result may be unexpected. It is a bug.

**Comment by dayAndnight2018** [comment]
Maybe there needs a horizonal scroll bar when input more words in the direction of horizon. Now is hidden in the tag "```"

**Comment by whistyun** [comment]
> what I want is just like this:
> 
> ```
> const(
>      A = A,
>      B = B
> )
> ```

I think you should use codeblock.
I will not remove indent-codeblock. And I think there is no way to cohabit indent-codeblock and more spaces in Markdown.

**Comment by dayAndnight2018** [comment]
> > what I want is just like this:
> > ```
> > const(
> >      A = A,
> >      B = B
> > )
> > ```
> 
> I think you should use codeblock.
> I will not remove indent-codeblock. And I think there is no way to cohabit indent-codeblock and more spaces in Markdown.

So, fix the indent of code block first.

**Comment by whistyun** [comment]
I released v0.10.1 and v0.9.1.
This version allows us to put codeblock in list.
And it has been confirmed to work with only one code block in the block quotes.

![image](https://user-images.githubusercontent.com/13712028/105976095-1c0e5780-60d3-11eb-8fbc-811548c4ebff.png)

**Comment by whistyun** [comment]
Note that an indented-codeblock syntax is changed.
In previous version, indented-codeblock didn't require a blank line at the beginning, but this version dose. 
This behavior is not mentioned in original spec, but is implemented.

![image](https://user-images.githubusercontent.com/13712028/105977775-203b7480-60d5-11eb-8239-c0a0b772b847.png)
